docs/src/Module.doc.js

```diff
@@ -12,49 +12,201 @@ card(
       {
         name: 'id',
+        href: 'non-expandable-default',
         type: 'string',
         required: true,
+        description: 'Id to identify this Module',
+      },
+      {
+        name: 'title',
+        href: 'non-expandable-default',
+        type: 'string',
+        description: 'Title of this Module',
+      },
+      {
+        name: 'icon',
+        href: 'non-expandable-icon',
+        type: 'string',
+        description: 'Name of icon to display in front of title',
+      },
+      {
+        name: 'iconAccessibilityLabel',
+        href: 'non-expandable-icon',
+        type: 'string',
+        description:
+          'Label to provide information about the icon used for screen readers',
+      },
+      {
+        name: 'type',
+        href: 'non-expandable-error',
+        type: '"info" | "error"',
+        defaultValue: 'info',
+        description:
+          'If set to `error`, displays error icon and changes title to red text',
+      },
+      {
+        name: 'children',
+        href: 'non-expandable-default',
+        type: 'React.Node',
+        description: 'Content to display underneath Module title',
+      },
+    ]}
+  />
+);
+
+card(
+  <PropTable
+    Component={Module.Expandable}
+    name="Expandable Module"
+    id="Expandable Module"
+    props={[
+      {
+        name: 'id',
+        href: 'expandable-default',
+        type: 'string',
+        required: true,
+        description: 'Base Id used to give each item in the Module a unique Id',
       },
       {
         name: 'accessibilityExpandLabel',
+        href: 'expandable-default',
         type: 'string',
         required: true,
+        description:
+          'Label used to communicate to screen readers which module will be expanded when interacting with the title button. Should be something clear, like "Expand Security Policies Module"',
       },
       {
         name: 'accessibilityCollapseLabel',
+        href: 'expandable-default',
         type: 'string',
         required: true,
+        description:
+          'Label used to communicate to screen readers which module will be collapsed when interacting with the title button. Should be something clear, like "Collapse Security Policies Module"',
       },
       {
         name: 'items',
+        href: 'expandable-items',
         type:
           'Array<{| title: string, icon?: $Keys<typeof icons>, summary?: Array<string>, type?: "info" | "error", iconAccessibilityLabel?: string, children: ?React.Node |}>',
         required: true,
+        description:
+          'Array of modules to display in a stack - only one item can be expanded at a time',
       },
     ]}
   />
 );
 
 card(
   <Example
-    name="Example"
+    name="Non-Expandable"
+    description={`A Module is a container that can hold any content, and can optionally have a \`title\` that describes the content inside. The default, non-expandable Module is used to display information that should always be visible.`}
```

Comment:
nit: just use a string prop to obviate the need to escape characters

Reply:
This is actually done to create <code> blocks out of the prop names (string prop will just show the tics instead)